apps/builder/app/builder/features/style-panel/sections/advanced/add.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState } from "react";
```

Comment:
would fit into single module 🤷

Reply:
it would, but its 100LOC with a bunch of utilities and imports, you would have to name everything based on a larger scope etc, I tend to prefer smaller modules that focus on one thing